### DISCUSSION TITLE
User-created excel file (use case) not able to load

### DISCUSSION THREAD

**Discussion by JunejaAmmy** [CURENT/andes#250] [Q&A] [answered]
I have created an excel file and tried to load it using method **andes.load**.

Below is the user-created file.
[dpf_full.xlsx](https://github.com/cuihantao/andes/files/7855590/dpf_full.xlsx)

Below is the screen shot of the error.
![Andes_load_error](https://user-images.githubusercontent.com/97614806/149167267-11534dfd-e093-4226-832b-03bdb63af09f.png)

Please guide us, how to work with user-created file.

**Comment by cuihantao** [comment, marked as answer]
You can start with an example excel file in any of the `andes/cases` directories. For example, `andes/cases/kundur/kundur_full.xlsx` shows that you will need `Bus`, `Line`, and others as separate spreadsheets.

**Reply by JunejaAmmy** [reply]
Thanks Cui,

I want to test with my own excel file. I have attached my file as well. Please guide me if that is possible to load as an example excel file in any of the andes/cases directories.

**Reply by cuihantao** [reply]
You will need to check the examples and fix the difference.

**Reply by cuihantao** [reply]
How to load an example Excel file in `andes/cases` directory can be found in the notebook examples at https://github.com/cuihantao/andes/tree/master/examples

**Reply by JunejaAmmy** [reply]
Do we need to keep the same format (like headers, input-data) as in example file?

**Reply by cuihantao** [reply]
You will need to have the same structure as any of the examples. That is, bus information on a separate `Bus` sheet, line information on a `Line` sheet, etc.
